include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -721,16 +758,19 @@ class operational_domain_impl
      * @param lyt SiDB cell-level layout that is simulated and compared to the given CDS.
      * @return All physically valid physical parameters and the excited state number.
      */
-    [[nodiscard]] operational_domain<parameter_point, uint64_t>
+    [[nodiscard]] sidb_simulation_domain<parameter_point, uint64_t>
     grid_search_for_physically_valid_parameters(Lyt& lyt) noexcept
```

Comment:
Is this the correct return type still with the new domain types available now?

Reply:
yes! This is the special case for ``physically_valid_parameters``.